bytes/bytes/config.py

```diff
@@ -2,41 +2,83 @@
 from pathlib import Path
 from typing import Optional
 
-from pydantic import BaseSettings
+from pydantic import AmqpDsn, BaseSettings, Field, PostgresDsn
 
 from bytes.models import EncryptionMiddleware, HashingAlgorithm, HashingRepositoryReference
 
 BASE_DIR = Path(__file__).parent.parent
 
 
 class Settings(BaseSettings):
-    secret: str
-    bytes_username: str
-    bytes_password: str
-    queue_uri: Optional[str]
-    log_cfg: Path = BASE_DIR / "dev.logging.conf"
-
-    bytes_db_uri: str
-    bytes_data_dir: Path = Path("/data")
-
-    bytes_log_file: str = "bytes.log"
-    access_token_expire_minutes: float = 15.0
-    bytes_folder_permission: str = "740"
-    bytes_file_permission: str = "640"
-
-    hashing_algorithm: HashingAlgorithm = HashingAlgorithm.SHA512
-
-    ext_hash_repository: HashingRepositoryReference = HashingRepositoryReference.IN_MEMORY
-    pastebin_api_dev_key: str = ""
-    rfc3161_provider: Optional[str]
-    rfc3161_cert_file: Optional[Path]
-
-    encryption_middleware: EncryptionMiddleware = EncryptionMiddleware.IDENTITY
-    kat_private_key_b64: str = ""
-    vws_public_key_b64: str = ""
-
-    span_export_grpc_endpoint: Optional[str]
-    bytes_metrics_ttl_seconds: int = 300
+    secret: str = Field(..., description="Secret key used for generating Bytes' API JWT")
+    username: str = Field(..., description="Username used for generating Bytes' API JWT")
+    password: str = Field(..., description="Password used for generating Bytes' API JWT")
+    queue_uri: AmqpDsn = Field("amqp://", description="KAT queue URI", env="QUEUE_URI")
+    log_cfg: Path = Field(BASE_DIR / "dev.logging.conf", description="Path to the logging configuration file")
+
+    db_uri: PostgresDsn = Field("postgresql://xx:xx@host:5432/bytes", description="Bytes Postgres DB URI")
```

Comment:
Why would we set invalid defaults for a field? Wouldn't it be better to get an error that the configuration field is not set, rather than an error that the database connection failed because of some invalid default?

Reply:
Fair point. Maybe a middle ground would be to have empty but required defaults, and provide an example? 

E.g. `db_uri: PostgresDsn = Field(..., description="Bytes Postgres DB URI", example="postgresql://xx:xx@host:5432/bytes")`

Reply:
I've changed it. Let me know what you think :)